### DISCUSSION TITLE
[Feature Request] MP3 Player in Reader

### DISCUSSION THREAD

**Discussion by MrCamel999** [spacecowboy/Feeder#238] [Ideas]
I get notified about a new podcast through the RSS feed, and it contains a hyperlink that says <fileName>.MP3, which opens in my browser. Is it possible at all to just have the mp3 load in Feeder so I can listen in the app or not? I understand this would require a ton of work probably, so I completely understand if this isn't planned and doesn't want to be worked on at all.

**Comment by MrCamel999** [comment]
Just realized I can attach things lol. Attached are screenshots of what I'm taking about.![Screenshot_20240313-125937.png](https://github.com/spacecowboy/Feeder/assets/44412967/438ef31c-8a1c-4c85-8288-b12427a76f4a)

![Screenshot_20240313-125949.png](https://github.com/spacecowboy/Feeder/assets/44412967/b2bf6de0-cd4f-4cc6-bd43-62d4e17f65c0)

**Comment by spacecowboy** [comment]
Yeah it's one of those things that would be cool to have.

Wouldn't be terrible to add after I complete work on the revamped text-to-speech functionality.
